tools/ci/before_install.sh

```diff
@@ -1,24 +1,16 @@
 set -v
 
-echo "gem: --no-ri --no-rdoc --no-document" > ~/.gemrc
-travis_retry gem install bundler -v ">= 1.11.1"
-
 if [[ -n "${GEM}" ]] ; then
   cd gems/${GEM}
 else
-  echo "1" > REGION
-  cp certs/v2_key.dev certs/v2_key
-  cp config/database.pg.yml config/database.yml
-  cp config/cable.yml.sample config/cable.yml
-  psql -c "CREATE USER root SUPERUSER PASSWORD 'smartvm';" -U postgres
-  export BUNDLE_WITHOUT=development
+  source $TRAVIS_BUILD_DIR/tools/ci/setup_vmdb_configs.sh
```

Comment:
TRAVIS_BUILD_DIR is specific to [travis](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables).  It looks like a similar value with gitlab would be [CI_PROJECT_DIR](https://docs.gitlab.com/ce/ci/variables/README.html).  I wonder if we should have the environment we're running in set a "normalized" variable such as MIQ_CI_BUILD_DIR so most of our tools/ci can move to more generic scripts that rely on the environment variables that govern the CI we're building on. 

Note, we already have `travis_retry` in here so we're pinned to travis as the CI vendor here.  Just thinking out loud.